plugins/woocommerce/includes/class-wc-frontend-scripts.php

```diff
@@ -492,6 +509,14 @@ private static function get_script_data( $handle ) {
 							'smoothHeight'   => true,
 							'directionNav'   => false,
 							'controlNav'     => 'thumbnails',
+							/**
+							 * Controls if manual controls for the single product flexslider should be enabled.
+							 *
+							 * @since 6.7.0
+							 *
+							 * @param bool $enable Enable manual controls.
+							 */
+							'manualControls' => apply_filters( 'woocommerce_single_product_nav_flexslider', get_theme_support( 'wc-product-gallery-slider-nav' ) ) ? '.flex-control-nav img' : null,
```

Comment:
I'm wondering if we need this hook; I'm thinking that `manualControls` can be easily overridden and set to either `null` or a different selector string via existing hook `woocommerce_single_product_carousel_options`.

More granular hooks like this one can definitely be a valid way to go, but it also increases the total public API surface area when we perhaps do not need to.